cmd/nodecmd/list.go

```diff
@@ -45,29 +45,34 @@ func list(_ *cobra.Command, _ []string) error {
 	sort.Strings(clusterNames)
 	for _, clusterName := range clusterNames {
 		clusterConf := clustersConfig.Clusters[clusterName]
-		ux.Logger.PrintToUser("Cluster %q (%s)", clusterName, clusterConf.Network.Kind.String())
 		if err := checkCluster(clusterName); err != nil {
 			return err
 		}
+		nodeIDs := []string{}
 		for _, cloudID := range clusterConf.GetCloudIDs() {
-			nodeConfig, err := app.LoadClusterNodeConfig(cloudID)
-			if err != nil {
-				return err
-			}
 			nodeIDStr := "----------------------------------------"
 			if clusterConf.IsAvalancheGoHost(cloudID) {
 				nodeID, err := getNodeID(app.GetNodeInstanceDirPath(cloudID))
 				if err != nil {
-					return err
+					ux.Logger.PrintToUser(logging.Yellow.Wrap("could not obtain node ID for nodes %s: %s"), cloudID, err)
+				} else {
+					nodeIDStr = nodeID.String()
```

Comment:
simpler `nodeIDs = append(nodeIDs, nodeID.String())` here and no need `nodeIDs = append(nodeIDs, nodeIDStr)` later

Reply:
it is needed so, for the default case